activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 # Make sure we're using pg high enough for type casts and Ruby 2.2+ compatibility
-gem "pg", "~> 0.18"
+gem "pg", ">= 0.18", "< 2.0"
```

Comment:
💩 
It been intended to use gemspec!

Reply:
pg or other database adapters have never been part of the gemspec, because this would force the user to install all of them.

Reply:
Oh, I understand. Someone have to take out the adapter to a separate gem with own dependency. Maybe `activerecord-pg`.